src/EFCore.PG/Extensions/NpgsqlServiceCollectionExtensions.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.EntityFrameworkCore.Query;
 using Microsoft.EntityFrameworkCore.Query.ExpressionTranslators;
 using Microsoft.EntityFrameworkCore.Query.ExpressionVisitors;
+using Microsoft.EntityFrameworkCore.Query.Internal;
```

Comment:
I'm not seeing any changes that use this... did something move upstream, or was this unintentional?

Reply:
Oh I see, which also explains why Relinq requires the alias just below this. Thanks for clarifying!